app/flatpak-builtins-utils.c

```diff
@@ -459,8 +459,9 @@ flatpak_resolve_matching_refs (const char   *remote_name,
 
   if (chosen == 0)
     {
+      g_autofree char *dir_name = flatpak_dir_get_name (dir);
       g_print (_("Similar refs found for ‘%s’ in remote ‘%s’ (%s):\n"),
-               opt_search_ref, remote_name, flatpak_dir_get_name (dir));
+               opt_search_ref, remote_name, dir_name);
```

Comment:
I think I have a commit in some PR to make flatpak_dir_get_name_cached() non-static. That would be nice to use here.